test/acceptance/features/steps/openshift.py

```diff
@@ -148,15 +194,24 @@ def expose_service_route(self, service_name, namespace):
         return re.search(r'.*%s\sexposed' % service_name, output)
 
     def get_route_host(self, name, namespace):
-        (output, exit_code) = self.cmd.run(f'oc get route {name} -n {namespace} -o "jsonpath={{.status.ingress[0].host}}"')
-        exit_code | should.be_equal_to(0)
+        output, exit_code = self.cmd.run(f'oc get route {name} -n {namespace} -o "jsonpath={{.status.ingress[0].host}}"')
+        exit_code | should.be_equal_to(0).desc(f"Getting route host exit code should be 0\nERROR: {output}")
```

Comment:
the discussion out come was to replace `should` calls with `assert`

Reply:
I have replaced should with assert with corresponding error message.